_sass/custom/custom.scss

```diff
@@ -132,3 +132,12 @@ h4,
   font-size: $font-size-3 !important;
   line-height: 1rem;
 }
+// Disable anchor links on h1 titles
+h1:hover > .anchor-heading {
+  pointer-events: none;
+  cursor: default;
+  display: none;
+  svg {
+    visibility: hidden;
+  }
```

Comment:
Does it work if you only set `display: none` here?

Reply:
Actually, it doesn't work well if you keep `:hover` in the rule, But seems to work well if you do just this: 

```css
h1 > .anchor-heading {
    display: none;
}
```

Reply:
> Does it work if you only set display: none here?

I tried that first, and it seemed to work, but the icon would briefly appear randomly anyway, so I figured: let's add as many rules as we can to make sure that the icon doesn't show up. But it's probably too much, you're right.

> Actually, it doesn't work well if you keep :hover in the rule.

I'll follow your advice and add your code snippet to the PR. Thanks. 🙏